scripts.js

```diff
@@ -0,0 +1,80 @@
+const mobileMenu = document.querySelector('.mobile-menu');
+const closeBtn = document.querySelector('.close-btn');
+const toggleMenu = document.querySelector('.navbar-toggler-icon');
+toggleMenu.addEventListener('click', () => {
+  mobileMenu.classList.remove('hidden');
+});
+closeBtn.addEventListener('click', () => {
+  mobileMenu.classList.add('hidden');
+});
+window.addEventListener('resize', () => {
+  if (window.innerHeight >= 768) {
+    mobileMenu.classList.add('hidden');
+  }
+});
+
+const speakersArray = [
+  {
+    name: 'Dan Ssekasiko',
+    university:
+      ' Professor  of Computer Science & Engineering at the University of Washington',
+    img: './images/speakers/pexels-igreja-dimensão-10295936.jpg',
+    biodata: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+  },
+  {
+    name: 'Agaba Nelson',
+    university:
+      'Professor of Computer Science & Engineering at the University of Washington ',
+    img: './images/speakers/pexels-pranav-digwal-32976.jpg',
+    biodata: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+  },
+  {
+    name: 'Davin',
+    university:
+      'Professor of Computer Science & Engineering at the University of Washington',
+    img: './images/speakers/product-school-DL-yyYDDNX4-unsplash.jpg',
+    biodata: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+  },
+  {
+    name: 'James Garry',
+    university:
+      'Professor of Computer Science & Engineering at the University of Washington',
+    img: './images/speakers/pexels-igreja-dimensão-10295936.jpg',
+    biodata: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+  },
+  {
+    name: 'Daniel Matama',
+    university:
+      'Professor of Computer Science & Engineering at the University of Washington',
+    img: './images/speakers/product-school-DL-yyYDDNX4-unsplash.jpg',
+    biodata: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+  },
+  {
+    name: 'Glory Mugizi',
+    university:
+      'Professor of Computer Science & Engineering at the University of Washington',
+    img: './images/speakers/pexels-pranav-digwal-32976.jpg',
+    biodata: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+  },
+];
+const featuredSpeakers = document.querySelector('.speakers-container');
+speakersArray.forEach((speaker) => {
+  const {
+    name, university, img, biodata,
+  } = speaker;
+  featuredSpeakers.innerHTML += `<div class="speakers">
+  <div>
+    <div class="speaker-image">
+      <img
+        src=${img}
+      />
+    </div>
+  </div>
+  <div class="speaker-bio">
+    <h6>${name}</h6>
+    <p class="speaker-education">${university}</p>
+    <p>${biodata}</p>
+  </div>
+</div>
+`;
```

Comment:
- [ ] As is the case in the original template, kindly make sure when the page loads the user can only see two speakers on the page. and the `SHOW MORE` button. once clicked they can be able to see the all speaker's lists.

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/210803349-56b06332-c973-43d8-8950-da8ab3f16983.png)


Original design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/210803256-a98293e0-f406-4b41-bb13-e5d7d5c22d2e.png)


`Hint: With your approach, you may consider splitting the speakers in two different arrays: the one to show the initial list and the other to show the remaining list once the button is clicked. You may consider putting the second render in a function which will be called once the button is cliked ` :smile: